lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -119,49 +139,61 @@ Ensure that students are comfortable shifting between representations before dis
 |@math{5^2}							| @math{5 \times 2}
 |@show{(coe  '(expt 5 2))}		| @show{(coe  '(* 5 2))}
 |===
+
 }
 
+@teacher{
 With a basic understanding of how exponents work, students are ready to discover how Circles of Evaluation can help them in analyzing more complicated expressions with exponents.
+}
 
 === Investigate
 
 @QandA{
-Look at the two Circles of Evaluation below:
 
 [.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
 |===
 
 |@show{(coe  '(* 9 (expt 3 2)))}		| @show{(coe  '(expt (* 9 3) 2))}
 |===
 
-@Q{How would you translate each of these Circles of Evaluation into verbal expressions? Why are their translations different?}
+@Q{How would you translate each of these Circles of Evaluation (above) into verbal expressions? Why are their translations different?}
 @A{The Circle of Evaluation on the left is "nine multiplied by three squared," while the Circle of Evaluation on the right is "find the product of nine and three, then square it." Their translations differ because the Circles of Evaluation are organized differently.}
+
 @Q{How would you translate each of these Circles of Evaluation into mathematical expressions?}
 @A{The Circle of Evaluation on the left is @math{9 \times 3^2} while the Circle of Evaluation on the right is @math{(9 \times 3)^2}.}
 }
 
+@teacher{
 Students will likely suggest @math{9 \times (3^2)} as a translation for the Circle of Evaluation on the right. This response is correct - but it is valuable for students to recognize that expressions with exponents are often written _without_ the parentheses. That said, the base and the exponent should always be placed within a Circle.
+}
 
 @lesson-point{
 In an expression with no grouping symbols, we evaluate the exponent before the other operations.
 }
 
+@teacher{
```

Comment:
Again, way too many teacher notes. Either remove this, or just start the subsequent lesson-instruction with "Let's practice with this important concept."